### PR TITLE
More template fields

### PR DESCRIPTION
**Pull request by alexpmay**

### CODE REVIEW

File: src/main/java/io/causallabs/mustache/Feature.java [hunk 1]
```diff
@@ -18,17 +18,45 @@ public interface Feature {
...
+
+    /**
+     * The attribute of this feature that is used as the persistent key. Only applicable to sessions
+     */
+    public WarehouseColumn getPersistentKey();
+
+    /**
+     * The attributes of this feature that are defined with @elapsed
+     */
+    public Collection<WarehouseColumn> getElapsedColumns();
```
Comment (jpalmucci) [inline src/main/java/io/causallabs/mustache/Feature.java:60]: Elapsed takes a parameter. So this is going to need to change when we extend it with more values.

Comment (jpalmucci) [reply src/main/java/io/causallabs/mustache/Feature.java:60]: NVM, I see what you did below

File: src/main/java/io/causallabs/mustache/WarehouseColumn.java [hunk 2]
```diff
@@ -24,4 +24,34 @@ public interface WarehouseColumn {
...
+    public FileLocation getFileLocation();
+
+    /** returns the feature that this was defined on */
+    public String getFeatureName();
+
+    public enum ElapsedType {
+        SESSION_END
+    };
+
+    /** returns the elapsed type (@elapsed directive) */
```
Comment (jpalmucci) [inline src/main/java/io/causallabs/mustache/WarehouseColumn.java]: Null if not elapsed? Add to comment?

**Review by jpalmucci — APPROVED** [review]

File: src/main/java/io/causallabs/mustache/WarehouseColumn.java (hunk 2 above)
Comment (alexpmay) [reply src/main/java/io/causallabs/mustache/WarehouseColumn.java]: added

**Pull request merged**